components/frontend/src/dashboard/MetricSummaryCard.js

```diff
@@ -69,20 +69,36 @@ export function MetricSummaryCard({ header, onClick, selected, summary, maxY })
     const dates = Object.keys(summary)
     const bbWidth = boundingBox.width ?? 0
     const bbHeight = boundingBox.height ?? 0
-    const label = (
-        <VictoryPortal x={bbWidth / 2} y={bbHeight / 2}>
-            <VictoryLabel
-                textAnchor="middle"
-                style={{ fontFamily: "Arial", fontSize: 12, fill: labelColor }}
-                text={nrMetricsLabel(sum(summary[dates[0]]))}
-            />
-        </VictoryPortal>
-    )
     const chartProps = {
         animate: animate,
         colors: colors,
+        events: [
+            {
+                childName: "all",
+                target: "data",
+                eventHandlers: {
+                    onClick: () => {
+                        return [
+                            {
+                                target: "labels",
+                                mutation: () => {
+                                    return { active: false } // Clear tooltip before filtering
+                                },
+                            },
+                        ]
+                    },
+                },
+            },
+        ],
         height: Math.max(bbHeight, 1), // Prevent "Failed prop type: Invalid prop range supplied to VictoryBar"
-        label: label,
+        label: (
+            <VictoryLabel
+                text={nrMetricsLabel(sum(summary[dates[0]]))}
+                textAnchor="middle"
+                x={bbWidth / 2}
```

Comment:
Is it possible to use `// 2`, so an int is always returned? Or is it not a problem for VictoryLabel?

Reply:
Javascript doesn't have a `//` operator. It starts a comment. And apparently floats are no problem for the VictoryLabel.